patches/@perf-profiler+reporter+0.8.1.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/node_modules/@perf-profiler/reporter/dist/src/index.d.ts b/node_modules/@perf-profiler/reporter/dist/src/index.d.ts
+index 2f84d84..14ae688 100644
+--- a/node_modules/@perf-profiler/reporter/dist/src/index.d.ts
++++ b/node_modules/@perf-profiler/reporter/dist/src/index.d.ts
+@@ -4,4 +4,6 @@ export * from "./reporting/Report";
+ export * from "./utils/sanitizeProcessName";
+ export * from "./utils/round";
+ export * from "./reporting/cpu";
++export * from "./reporting/ram";
```

Comment:
For that I created PR to upstream: https://github.com/bamlab/flashlight/pull/294